.goreleaser.yml

```diff
@@ -40,3 +40,16 @@ nfpm:
   formats:
     - deb
     - rpm
+
+brew:
+  name: "exo"
+  github:
+    owner: exoscale
+    name: homebrew-tap
+  folder: Formula
+  homepage: "https://exoscale.github.io/cli/"
+  description: Manage (almost) everything at Exoscale from the command line.
+  dependencies:
+      - go
+  skip_upload: true
```

Comment:
- `exo` itself has no dependencies, we can skip this section
- we can also let it upload the Formula as it'll only be done when a release is made